lib/buildWordsArray.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = function buildWordsArray(wordObject1, wordObject2, wordObject3) {
+  var adjectivesArray = objectToString(wordObject1);
+  var nounsArray = objectToString(wordObject2);
+  var verbsArray = objectToString(wordObject3);
+
+  console.log(adjectivesArray);
+  return '<tr><th id="tableColumn">Adjectives</th></tr>' + adjectivesArray +
+  '<tr><th id="tableColumn">Nouns</th></tr>' + nounsArray +
+  '<tr><th id="tableColumn">Verbs</th></tr>' + verbsArray;
```

Comment:
cool! You can also just return the words here, and then use your jQuery to add on those html tags.